examples/CInterface/CMakeLists.txt

```diff
@@ -69,7 +69,17 @@ target_compile_options(hello_world_sender PRIVATE -ansi -Wall -pedantic -Werror)
 target_compile_options(hello_world_receiver PRIVATE -ansi -Wall -pedantic -Werror)
 endif()
 
-
+if (BUILD_HELICS_TESTS AND NOT MSVC)
```

Comment:
this will need to check for BUILD_HELICS_EXAMPLES as well (check that cmake flag)  those are separate options and both need to be enabled for these tests to make sense